# 00_prefix: by Ryan Kavanagh <ryanakca@kubuntu.org>
# Description: Sets the prefix for the Turnin-NG installation
Index: Turnin-NG/setup.py
===================================================================
--- Turnin-NG.orig/setup.py	2010-02-12 07:31:06.676104825 -0500
+++ Turnin-NG/setup.py	2010-02-12 14:10:27.807856407 -0500
@@ -13,7 +13,7 @@
 import tempfile
 
 # Default prefix
-prefix = '/usr/local'
+prefix = ''
 # Get the install prefix if one is specified from the command line
 for i, arg in enumerate(sys.argv):
     prefix_regex = re.compile('(?P<prefix>--prefix)?[\=\s]?(?P<path>/[\w\s/]*)')
